### PR TITLE
Update the references of Ubuntu versions for SITARA platforms

### PR DESCRIPTION
**Pull request by jeevantelukula**
- Update the supported ubuntu version to 22.04
- Remove the reference of `How to Build a Ubuntu Linux host under VMware` since it is no longer maintained

### CODE REVIEW

**Comment by cshilwant** [comment]
@jeevantelukula The workflow throws an error. Please check failures reported by bots - https://github.com/TexasInstruments/processor-sdk-doc/actions/runs/11700214328?pr=34

Refer https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md for more details on CONTRIBUTING guidelines

File: source/devices/AM335X/linux/Release_Specific_Release_Notes.rst [hunk 1]
```diff
@@ -309,8 +309,7 @@ Composer Studio.
 .. rubric:: Host Support
    :name: host-support
 
-The Processor SDK is developed, built and verified on Ubuntu 22.04. Details on how to create a virtual machine to load Ubuntu
-are described in :ref:`this page <how-to-build-a-ubuntu-linux-host-under-vmware>`.
```
Comment (cshilwant) [inline source/devices/AM335X/linux/Release_Specific_Release_Notes.rst:313]: Well the section "How to Build a Ubuntu Linux host under VMware" [1] is no longer maintained & carries stale information.
 
IMO, we should delete the entire "How_to_Build_a_Ubuntu_Linux_host_under_VMware" file & remove references of it from all the <SOC>_linux_toc tree as well

[1]: https://software-dl.ti.com/processor-sdk-linux/esd/AM65X/09_01_00_01/exports/docs/linux/How_to_Guides/Host/How_to_Build_a_Ubuntu_Linux_host_under_VMware.html

**Review by cshilwant — CHANGES_REQUESTED** [review]
Needs Work

File: source/devices/AM335X/linux/Release_Specific_Release_Notes.rst (hunk 1 above)
Comment (jeevantelukula) [reply source/devices/AM335X/linux/Release_Specific_Release_Notes.rst:313]: This file has been used for devices other than those in the SITARA platform. Currently, its references have been removed from all the _linux_toc tree for SITARA devices

File: source/devices/AM65X/linux/Release_Specific_Release_Notes.rst [hunk 2]
```diff
@@ -350,8 +350,7 @@ Composer Studio.
 Host Support
 ============
 
-The Processor SDK is developed, built and verified on Ubuntu 16.04 and 18.04. Details on how to create a virtual machine to load Ubuntu
-are described in :ref:`this page <how-to-build-a-ubuntu-linux-host-under-vmware>`.
+The Processor SDK is developed, built and verified on Ubuntu 22.04.
```
Comment (cshilwant) [inline source/devices/AM65X/linux/Release_Specific_Release_Notes.rst:353]: @jeevantelukula  Better to use the macro in all places. 

```suggestion
The Processor SDK is developed, built and verified on  Ubuntu |__LINUX_UBUNTU_VERSION_SHORT__|.
```

Note - You might also have to add `.. include:: /_replacevars.rst` 

Check - https://github.com/TexasInstruments/processor-sdk-doc/blob/master/source/linux/Overview/Processor_SDK_Linux_Getting_Started_Guide.rst?plain=1#L1

Comment (jeevantelukula) [reply source/devices/AM65X/linux/Release_Specific_Release_Notes.rst:353]: When built locally, it shows the accurate version even without including `replacevars.txt`

**Review by cshilwant — APPROVED** [review]

File: bin/rst_wiki_list_linux.txt [hunk 3]
```diff
@@ -172,8 +172,6 @@ source/linux/How_to_Guides/Host/Erase_Flash_for_AM335x_ICE_v2.rst
 http://processors.wiki.ti.com/index.php/AM335x_ICEv2_flash_erase
 source/linux/How_to_Guides/Host/Understand_and_Build_Boot_Monitor_for_Keystone2.rst
 http://processors.wiki.ti.com/index.php/Linux_Core_Boot_Monitor_User%27s_Guide
-source/linux/How_to_Guides/Host/Build_a_Ubuntu_Linux_host_under_VMware.rst
-http://processors.wiki.ti.com/index.php/How_to_Build_a_Ubuntu_Linux_host_under_VMware
```
Comment (StaticRocket) [inline bin/rst_wiki_list_linux.txt:176]: This ref should not be removed if the file still exists.

File: source/devices/AM65X/linux/Release_Specific_Release_Notes.rst (hunk 2 above)
Comment (StaticRocket) [reply source/devices/AM65X/linux/Release_Specific_Release_Notes.rst:353]: This was done for `source/linux/Overview/Processor_SDK_Linux_Getting_Started_Guide.rst`, but not for any of the device specific release notes. You should use `|__LINUX_UBUNTU_VERSION_SHORT__|` in those instances as well.

**Review by StaticRocket — CHANGES_REQUESTED** [review]

**Review by StaticRocket — DISMISSED** [review]

**Review by cshilwant — APPROVED** [review]

**Review by StaticRocket — APPROVED** [review]

**Review by praneethbajjuri — APPROVED** [review]

**Pull request merged**